### PR TITLE
VPN + Bastion LogDNA

### PR DESCRIPTION
**Pull request by aaahrens**
Example triggers are in the slack. Should they be part of the readme? 

Making another branch for cloudwatch and another for the LogDNA Daemonset.

### CODE REVIEW

File: packer/image.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             "instance_type": "c5.large",
             "region": "{{user `region`}}",
             "source_ami": "{{user `source_ami`}}",
-            "ssh_username": "ec2-user",
+            "ssh_username": "centos",
```
Comment (abhiyerra) [inline packer/image.json]: Hmm this doesn't seem to be needed?

File: eks/vpn.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 resource "aws_eip" "vpn_eip" {
   instance = aws_instance.vpn[0].id
   vpc      = true
-  count       = var.foxpass_api_key != "" ? 1 : 0
+  count    = var.foxpass_api_key != "" ? 1 : 0
```
Comment (abhiyerra) [inline eks/vpn.tf]: Why is this here? It should maae it regardless?

Comment (aaahrens) [reply eks/vpn.tf]: It doesn't make a vpn available if the foxpass api key isnt set. Should I change it? I though the normal ipv4 of an ec2 instance isn't mapped to a vpc.

Comment (abhiyerra) [reply eks/vpn.tf]: Oh I see, Sorry was thinking LogDNA for some reason.
